src/main/java/com/facebook/openwifirrm/ucentral/models/AggregatedState.java

```diff
@@ -169,11 +169,6 @@ public AggregatedState(
 		this.radio = new Radio(radioInfo);
 	}
 
-	@Override
-	public int hashCode() {
```

Comment:
It is good practice to override `hashCode` and `equals` or to override neither. I think the solution is to implement `hashCode` (using Eclipse's automatic tool to do this) and just not use it... Alternatively, if `equals` is never used, we could remove both `hashCode` and `equals`

Jeffrey mentioned they should not differ in which variables they check as well. (Actually, I don't see a problem with `hashCode` checking _fewer_ variables than equals, since AFAIK we only must guarantee that objects that are `equal` have the same `hashCode`, and we don't care about the other direction. That being said, maybe I'm missing something, and maybe it is more confusing to have `hashCode` look at fewer variables, so I'm fine with the idea of keeping them consistent.)

Reply:
As it's not used at this time, I will delete it.